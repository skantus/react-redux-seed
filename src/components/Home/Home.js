import React from 'react';
import { string } from 'prop-types';
import { Grid } from 'react-flexbox-grid';
import { content } from './home.module.css';

const Home = ({ tech }) => (
  <Grid fluid className={content}>
    <div>Tech: {tech}</div>
  </Grid>
);

Home.propTypes = {
  tech: string.isRequired
};

export default Home;
