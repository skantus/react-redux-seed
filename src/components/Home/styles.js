export const styles = theme => ({
  button: {
    margin: theme.spacing.unit * 2
  }
});
