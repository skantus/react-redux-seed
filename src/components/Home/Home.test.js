import React from 'react';
import Home from './Home';

describe('<Home />', () => {
  const wrapper = shallow(<Home tech="react" />);

  it('render <Home /> component as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
