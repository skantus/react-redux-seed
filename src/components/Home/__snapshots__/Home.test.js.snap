// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> render <Home /> component as expected 1`] = `
<Grid
  className="content"
  fluid={true}
>
  <div>
    Tech: 
    react
  </div>
</Grid>
`;
